### PR TITLE
MNIST tutorial incorrectly described a file

### PR DESCRIPTION
**Pull request by MicaelCarvalho**
The file "input_data.py" from MNIST tutorial does not download any data, as previously stated.

Related to #2733.

### CODE REVIEW

File: tensorflow/g3doc/tutorials/mnist/pros/index.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ TensorFlow session.
 
 For your convenience, we've included
 [a script](https://www.tensorflow.org/code/tensorflow/examples/tutorials/mnist/input_data.py)
-which automatically downloads and imports the MNIST dataset. It will create a
-directory `'MNIST_data'` in which to store the data files.
+which will help you download and import the MNIST dataset. Run the following commands to create a
+directory `'MNIST_data'` in the current folder, the data files will be stored inside of such directory.
```
Comment (vrv) [inline tensorflow/g3doc/tutorials/mnist/pros/index.md]: "will be stored inside that directory"

**Comment by MicaelCarvalho** [comment]
Text updated according to @vrv comment.

**Comment by maciekcc** [comment]
Jenkins, test this please.

**Comment by maciekcc** [comment]
Jenkins, test this please.

**Pull request merged**